phpstan.neon

```diff
@@ -0,0 +1,23 @@
+includes:
+    - ./phpstan-baseline.neon
+    - vendor/larastan/larastan/extension.neon
+    - vendor/phpstan/phpstan-mockery/extension.neon
+    - vendor/phpstan/phpstan-phpunit/extension.neon
+parameters:
+    level: 0
+    paths:
+        - ./config
+        - ./database
+        - ./src
+        - ./tests
```

Comment:
In my opinion I don't see the need for tests folder

Reply:
I rather analyse everything possible, some type mismatch or other issue can generate incorrect test results.